overrides/groovy/postInit/Post-Initial/Main/General/Mid-Game/midgame.groovy

```diff
@@ -2,6 +2,18 @@ import static com.nomiceu.nomilabs.groovy.GroovyHelpers.RecipeRecyclingHelpers.*
 
 // Processing Array
 replaceRecipeInput(metaitem('processing_array'),
-	[[ore('circuitLuv'), item('advsolars:sunnarium'), ore('circuitLuv')],
-	 [metaitem('robot.arm.iv'), metaitem('hull.iv'), metaitem('robot.arm.iv')],
-	 [ore('circuitLuv'), metaitem('tool.datastick'), ore('circuitLuv')]])
+        [[ore('circuitLuv'), item('advsolars:sunnarium'), ore('circuitLuv')],
+         [metaitem('robot.arm.iv'), metaitem('hull.iv'), metaitem('robot.arm.iv')],
+         [ore('circuitLuv'), metaitem('tool.datastick'), ore('circuitLuv')]])
+
+// PBI ingots into block
+def ingotPolybenzimidazole = ore('ingotPolybenzimidazole')
+def blockPolybenzimidazole = metaitem('blockPolybenzimidazole')
+def inputList1 = [ingotPolybenzimidazole] * 9 // Create a list with 9 instances of ingotPolybenzimidazole
```

Comment:
Just change this into `crafting.addShapeless(metaitem('blockPolybenzimidazole'), [ore('ingotPolybenzimidazole') * 9])`. Get rid of the variables. Same with below.

Reply:
It broke without the variables